tox.ini

```diff
@@ -7,10 +7,11 @@ deps =
     coverage
 setenv = PYTHONDEVMODE = 1
 commands =
-    coverage run --concurrency=multiprocessing -m pytest --doctest-modules --pyargs xopen tests
+    coverage run -m pytest --doctest-modules tests
     coverage combine
```

Comment:
`coverage combine` is only necessary when running with `--concurrency=multiprocessing` or `-p`, so this can be removed as well (if `parall = True` is also removed from the configuration).

Reply:
Ah, I was wondering why my usual coverage fu was not working. 
I removed setup.cfg and updated tox.ini accordingly.